docs/source/commandline/commandline-all.rst

```diff
@@ -27,17 +27,20 @@ short name (e.g. **a2600**, **zorba_kbd**) or a glob pattern that matches either
 Depending on the command you're using the pattern with, pattern matching may
 match systems or systems and devices.  It is advised to put quotes around your
 patterns to avoid having your shell try to expand them against filenames (e.g.
-**mame -validate "pac\*"**).
+**mame -validate "pac\*"**). This is true on all shells on Linux or Mac. On
+Windows you'll find that while cmd.exe doesn't need the quotes, PowerShell
+needs the quotes around patterns.
```

Comment:
The quotes are harmless for cmd.exe though, and you can run MAME with a shell that does glob substitution on Windows as well via MinGW, cygwin, etc.  Does the extra text help, given adding quotes is harmless on the one shell that doesn’t do glob substitution?

Reply:
Most CMD users don't think about quoting outside of filenames with spaces. Powershell is still foreign to a lot of Windows users. Nevertheless, it's not our place to teach them how to use PS, so this is being removed.